lms/static/sass/views/_login-register.scss

```diff
@@ -13,10 +24,24 @@
     $third-party-button-height: ($baseline*1.75);
 
     h2 {
-        @extend %t-title5;
+        @extend %t-title4;
+        margin: 0;
+        letter-spacing: normal;
+        font-family: $sans-serif;
+        color: $uxpl-blue-hover-active;
+    }
+
+    h3 {
+        @extend %t-title6;
+        @extend %t-light;
         margin: 0;
         letter-spacing: normal;
         font-family: $sans-serif;
+        color: #111111;
```

Comment:
Should this be a fixed color, or use one of the named colors in the edX pattern guide?